contracts/connectors/loantoken/LoanTokenLogicBeacon.sol

```diff
@@ -0,0 +1,143 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "../../mixins/EnumerableBytes32Set.sol";
+import "../../mixins/EnumerableBytes4Set.sol";
+import "../../openzeppelin/PausableOz.sol";
+import "../../openzeppelin/Address.sol";
+
+/**
+ * @title Loan Token Logic Beacon contract.
+ *
+ * @notice This contract stored the target logic implementation of LoanTokens which has the same logic implementation (LoanTokenLogicLM / LoanTokenLogicWrbtc)
```

Comment:
* @notice This contract stores the target logic implementation of LoanTokens which has the